bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -130,12 +130,16 @@ func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, re
 
 	// Look up the owner of this KubeConfig if there is one
 	configOwner, err := bsutil.GetConfigOwner(ctx, r.Client, config.ObjectMeta)
+	if apierrors.IsNotFound(err) {
+		// Could not find the owning machine yet, this is actually ok and we just need to wait a second
```

Comment:
```suggestion
		// Could not find the owner yet, this is actually ok and we just need to wait a second
```

Given that this is either Machine or Pools

Reply:
good call